pkg/unikontainers/unikontainers.go

```diff
@@ -494,7 +505,7 @@ func (u *Unikontainer) executeHook(hook specs.Hook, state []byte, wg *sync.WaitG
 		"path": hook.Path,
 		"args": hook.Args,
 		"env":  hook.Env,
-	}).Info("executing hook")
+	}).Error("executing hook")
```

Comment:
Let's not use Error logs for non-error cases.

Reply:
I have changed the log level.